src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -23,6 +23,12 @@ To start/stop user's local video, use `toggleVideo` function from the `LocalVide
   // A function to set the value of `isVideoEnabled`
   setIsVideoEnabled: (isEnabled: boolean) => void;
 
+  // Whether or not video limit is reached, it defaults to false
+  hasReachedVideoLimit: boolean;
+
+  // A function to set the value of `hasReachedVideoLimit`
+  setHasReachedVideoLimit: (hasReachedVideoLimit: boolean) => void;
```

Comment:
Can we remove this? It is not used right?

Reply:
yes, correct. Will remove it